oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentNodeStoreServiceTest.java

```diff
@@ -203,42 +189,6 @@ public void persistentCacheExclude() throws Exception{
         assertFalse(dns.getNodeCachePredicate().apply(Path.fromString("/x")));
     }
 
-    @Test
-    public void preset() throws Exception {
-        MockOsgi.setConfigForPid(context.bundleContext(),
-                Configuration.PRESET_PID,
-                DocumentNodeStoreServiceConfiguration.PROP_SO_KEEP_ALIVE, true);
-        MockOsgi.activate(preset, context.bundleContext());
-
-        MockOsgi.setConfigForPid(context.bundleContext(), PID, newConfig(repoHome));
-        MockOsgi.activate(service, context.bundleContext());
-
-        DocumentNodeStore store = context.getService(DocumentNodeStore.class);
-        MongoDocumentStore mds = getMongoDocumentStore(store);
-        assertNotNull(mds);
-        MongoClient client = MongoDocumentStoreTestHelper.getClient(mds);
-        assertTrue(client.getMongoClientOptions().isSocketKeepAlive());
-    }
-
-    @Test
-    public void presetOverride() throws Exception {
-        MockOsgi.setConfigForPid(context.bundleContext(),
-                Configuration.PRESET_PID,
-                DocumentNodeStoreServiceConfiguration.PROP_SO_KEEP_ALIVE, true);
-        MockOsgi.activate(preset, context.bundleContext());
-
-        Map<String, Object> config = newConfig(repoHome);
-        config.put(DocumentNodeStoreServiceConfiguration.PROP_SO_KEEP_ALIVE, false);
-        MockOsgi.setConfigForPid(context.bundleContext(), PID, config);
-
-        MockOsgi.activate(service, context.bundleContext());
-
-        DocumentNodeStore store = context.getService(DocumentNodeStore.class);
-        MongoDocumentStore mds = getMongoDocumentStore(store);
-        MongoClient client = MongoDocumentStoreTestHelper.getClient(mds);
-        assertFalse(client.getMongoClientOptions().isSocketKeepAlive());
-    }
-
```

Comment:
Can you please re-introduce these tests again and use another configuration that is still available to exercise the configuration preset feature?

Reply:
Yes, I have re-introduce both tests using the property PROP_UPDATE_LIMIT